### PR TITLE
Sscs 6057 case updated notification shouldnotscheduled

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java [hunk 1]
```diff
@@ -198,13 +198,15 @@ private String getSubscriptionDetails(String newSubscription, String oldSubscrip
...
         boolean isAllowed = true;
-        if (NotificationEventType.REQUEST_INFO_INCOMPLETE.equals(notificationType)) {
+        if (REQUEST_INFO_INCOMPLETE.equals(notificationType)) {
             if (StringUtils.isEmpty(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())
                     || "No".equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())) {
                 isAllowed = false;
 
                 log.error("Request Incomplete Information with empty or no Information From Appellant for ccdCaseId {}.", notificationWrapper.getNewSscsCaseData().getCcdCaseId());
             }
+        } else if (CASE_UPDATED.equals(notificationType)) {
```
Comment (ryandac) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java]: Does this stop all Case updated notifications from being sent out?

File: src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java [hunk 2]
```diff
@@ -2039,7 +2172,11 @@ private void verifyExpectedLogErrorCount(Appender<ILoggingEvent> mockAppender, A
         );
         List<ILoggingEvent> logEvents = (List<ILoggingEvent>) captorLoggingEvent.getAllValues();
         if (expectedErrors == 0) {
-            assertFalse(logEvents.stream().noneMatch(e -> e.getLevel().equals(Level.ERROR)));
+            if (logEvents.isEmpty()) {
+                assertEquals(logEvents.size(), expectedErrors);
```
Comment (ryandac) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java]: not sure if this change is needed. Isn't this the same as `assertFalse(logEvents.stream().noneMatch(e -> e.getLevel().equals(Level.ERROR)));` ?

File: src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java (hunk 1 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java]: yes, at the moment we don't have template for case updates and case updates is internal event. we don't want to notify to user

File: src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java (hunk 2 above)
Comment (srochani) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java]: If logevents size is 0 then assertFalse(logEvents.stream().noneMatch(e -> e.getLevel().equals(Level.ERROR))) equals will not be called and condition won't meet

Comment (srochani) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java]: It was failing test on this point

File: src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java (hunk 1 above)
Comment (ryandac) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java]: Ok. It should also be removed from the ccd spreadsheet at some point too.

**Pull request merged**